configure.ac

```diff
@@ -5083,6 +5083,9 @@ AC_CHECK_FUNCS(setpgrp,
     [])
 )
 
+# check for namespace functions
+AC_CHECK_FUNCS([setns unshare])
```

Comment:
The Linux manpage says that the function comes from ``sched.h``. Don't you have to include it explicitly? Something like (test for ``setgroups`` coming from ``<unistd.h>``):

```
# On some systems, setgroups is in unistd.h, on others, in grp.h
PY_CHECK_FUNC([setgroups], [
#include <unistd.h>
#ifdef HAVE_GRP_H
#include <grp.h>
#endif
])
```

You may use:

```
#ifdef HAVE_SCHED_H
#  include <sched.h>
#endif
```

I don't know well configure. @erlend-aasland: what do you think?

Reply:
`AC_CHECK_FUNCS` only make sure linking succeeded (`ac_fn_c_try_link`), so headers are not necessary.
I don't see headers are added to [most `AC_CHECK_FUNCS`](https://github.com/python/cpython/blob/50c48098f21be3e2648c55935fad5adb82319ea7/configure.ac#L4782)

Reply:
Yes, `AC_CHECK_FUNCS` only checks if a symbol is available from a library; headers are not needed.